shared/src/main/scala/zeno/examples/MultiPaxos.proto

```diff
@@ -0,0 +1,108 @@
+syntax = "proto2";
+
+import "scalapb/scalapb.proto";
+
+option (scalapb.options) = {
+  package_name: "zeno.examples"
+  flat_package: true
+};
+
+// Incoming client requests containing command they want to get chosen
+message ClientRequest {
+  required string command = 1;
+}
+
+// Messages sent between replicas and leaders.
+message ProposeToLeader {
+  required int32 slot = 1;
+  required string command = 2;
+}
+
+// Messages sent to decide command for a slot
+message Decision {
+  required int32 slot = 1;
+  required string command = 2;
+}
+
+// Struct holding the ballot, slot, and command for a given proposal
+message ProposedValue {
```

Comment:
I think `ProposedValue` is serving two different purposes in Phase 1b and Phase 2a. In Phase 1b, it looks like `ballot` is the vote round, `slot` is the slot, and `command` is the vote value. But in Phase 2a, `ballot` is just the ballot, `slot` is the slot, and `command` is the command being proposed. I would split them up.